styling.md

```diff
@@ -3,27 +3,27 @@ _The original version of this spec can be found at [chriskempson/base16](https:/
 ---
 
 # Base16 Styling Guidelines
-**Version 0.4.2**
+**Version 0.4.3**
 
-Base16 aims to group similar language constructs with a single color. For example, floats, ints, and doubles would belong to the same colour group. The colors for the default scheme were chosen to be easily separable, but scheme designers should pick whichever colours they desire, e.g. `base0B` (green by default) could be replaced with red. There are, however, some general guidelines below that stipulate which `base0B` should be used to highlight each construct when designing templates for editors.
+Base16 aims to group similar language constructs with a single color. For example, floats, ints, and doubles would belong to the same colour group. The colors for the default scheme were chosen to be easily separable, but scheme designers should pick whichever colors they desire, e.g. `base0B` (green by default) could be replaced with red. There are, however, some general guidelines below that stipulate which `base0B` should be used to highlight each construct when designing templates for editors.
 
-Since describing syntax highlighting can be tricky, please see [base16-vim](https://github.com/tinted-theming/base16-vim/) and [base16-emacs](https://github.com/tinted-theming/base16-emacs/) for reference. Though it should be noted that each editor will have some discrepancies due the fact that editors generally have different syntax highlighting engines.
+Since describing syntax highlighting can be tricky, please see [tinted-vim](https://github.com/tinted-theming/tinted-vim/) and [base16-emacs](https://github.com/tinted-theming/base16-emacs/) for reference. Though it should be noted that each editor will have some discrepancies due the fact that editors generally have different syntax highlighting engines.
 
-Colors `base00` to `base07` are typically variations of a shade and run from darkest to lightest for dark themes. These colors are used for foreground and background, status bars, line highlighting and such. Colors `base08` to `base0F` are typically individual colors used for types, operators, names and variables. In order to create a dark scheme, colors `base00` to `base07` should span from dark to light. For a light scheme, these colours should span from light to dark.
+Color properties `base00` to `base07`, with a prefixed hash `#` value, are typically variations of a shade and run from darkest to lightest for dark themes. These colors are used for foreground and background, status bars, line highlighting and such. Colors `base08` to `base0F` are typically individual colors used for types, operators, names and variables. In order to create a dark scheme, colors `base00` to `base07` should span from dark to light. For a light scheme, these colors should span from light to dark.
```

Comment:
I personally think we don't need to specify this here, as it's already noted in builder.md under the [common scheme format section](https://github.com/tinted-theming/home/blob/2c345896766b028064e982bd8d6f1a9b1526cbd6/builder.md#schemes).

Reply:
I mostly just wanted to fix the example and thought I should update it here too, but updating the example alone works!